internal/logstore/lokistack/logstore_lokistack.go

```diff
@@ -24,8 +24,9 @@ const (
 	lokiStackWriterClusterRoleName        = "logging-collector-logs-writer"
 	lokiStackWriterClusterRoleBindingName = "logging-collector-logs-writer"
 
-	lokiStackAppReaderClusterRoleName        = "logging-application-logs-reader"
-	lokiStackAppReaderClusterRoleBindingName = "logging-all-authenticated-application-logs-reader"
+	lokiStackAppViewClusterRoleName   = "cluster-logging-application-view"
```

Comment:
This will break existing configurations depending on the name of the ClusterRole. Do we consider the previous name part of our "public API" of the cluster-logging-operator? If yes, we need to keep the old name. If no, we can change it, but need to be aware of that change.

The old ClusterRole/ClusterRoleBinding will stay in the cluster during an upgrade, so the functionality will not be impacted, this is something that will need to be documented though.

Reply:
How would this break the existing configuration?

Reply:
I'm thinking of customers that have created their own RoleBindings/ClusterRoleBindings to our ClusterRole. That's why I'm asking if we're considering the role name part of our public API.

Reply:
IIUC the role will still be there after the update, so no breaking in behavior, but we will need to properly document this change.